tensorboard/webapp/metrics/data_source/metrics_data_source.ts

```diff
@@ -179,8 +188,26 @@ export class TBMetricsDataSource implements MetricsDataSource {
         })
       );
     });
+    const isImagesSupported$ = this.store.select(getIsFeatureFlagsLoaded).pipe(
```

Comment:
I think what you've written here using both getIsFeatureFlagsLoaded and getIsMetricsImageSupportEnabled is most correct and future proof, but a few thoughts:

* The way feature flag values are currently loaded, we have been able to assume elsewhere in the code that it is safe to access feature flag selectors without first checking getIsFeatureFlagsLoaded. But this only holds true if (1) FeatureFlagModule is properly imported before your code runs and (2) fetch of feature flag values occurs synchronously when FeatureFlagEffects is loaded (which it does right now). 
  * https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/feature_flag/effects/feature_flag_effects.ts;l=39;drc=9b29bb2706e5443052ac5fefacab742dfda11192).
* I hadn't really thought before about why this worked and why it might not work in the future. Again I think what you've written here is probably the most correct and most future proof.
* On the other hand, what you've written here is kind of verbose and its a shame that accessing a feature flag requires this sort of verbosity.
* Would it be worth adding some sort of helper function that we can use to access FeatureFlag values that hides this complexity?

  ```
  export function getFeatureFlag(store : State<FeatureFlagState>, featureFlagSelector : ??) 
      : Observable<boolean>
  ```

  The observable it returns would only emit a single value.

I'm not suggesting that you do that in this PR or that you do it at all. I mostly want to know if you think this chain of thought is agreeable. If so I may volunteer to do this cleanup, time permitting.